x-pack/plugins/fleet/server/services/package_policy.ts

```diff
@@ -1125,6 +1125,7 @@ class PackagePolicyClientImpl implements PackagePolicyClient {
     }
 
     // Bump revision of all associated agent policies (old and new)
+    const isEndpointPolicy = newPolicy.package?.name === 'endpoint';
```

Comment:
nit: although it seems to work in the same way as the version on `main`, i think it would help future maintenance (we will still have 2 new versions for 8.17) to put this line above line 1133, so the two modifications are more similar